Example/DeconvEx/2D/Deconv_LS_TV_NonNeg.m

```diff
@@ -64,7 +64,7 @@
 ADMM=OptiADMM(F,Fn,Hn,rho_n);
 ADMM.OutOp=OutputOptiSNR(1,im,10,[1 2]);
 % STOP when the sum successives C = F*x + Fn{1}*Hn{1}*x is lower than 1e-4 or when the distance between two successive step is lower than 1e-5
-ADMM.CvOp=TestCvgCombine(TestCvgCostRelative(1e-4,[1 2]), 'StepRelative',1e-4);  
+ADMM.CvOp=TestCvgCombine(TestCvgCostRelative(1e-4,[1 2]), 'StepRelative',1e-4,TestCvgADMM());  
```

Comment:
The test of `TestCvgADMM()` is working (even without `TestCvgCostRelative` and `StepRelative`

Reply:
:+1: Perfect ! Can merge :)
Thanks !